package.json

```diff
@@ -55,8 +57,10 @@
     "@babel/plugin-transform-parameters": "^7.1.0",
     "@babel/preset-env": "^7.1.0",
     "@babel/preset-react": "^7.0.0",
+    "@babel/preset-typescript": "^7.6.0",
     "@babel/register": "^7.0.0",
     "@emotion/babel-preset-css-prop": "10.0.9",
+    "@rollup/plugin-typescript": "^2.0.1",
```

Comment:
This has a `peerDependency` of `"rollup": "^1.20.0",`. We're still on v0. I think we should install an older version of this (it looks like `0.8.1` is the last version before `^1`, which requires more rollup changes.)

If it is easier, it might be preferable to spend cooldown working on updating the build dependencies in here in a separate PR, if you find the older versions are blocking in some regard (they apply to much older versions of TypeScript...)

Reply:
done, Yeah this was when I was trying to make `shared-component` work with TS I reverted back to its initial plugin compatible with older rollup